src/docfx/config/ContributionConfig.cs

```diff
@@ -37,5 +39,10 @@ internal class ContributionConfig
         /// It should be an absolute url or a relative path
         /// </summary>
         public readonly string GitCommitsTime = string.Empty;
+
+        /// <summary>
+        /// The excluded contributors which you do want to show
+        /// </summary>
+        public readonly string[] ExcludedContributors = Array.Empty<string>();
```

Comment:
just `exclude` since it is already under `contribution` node?

Reply:
Contribution is not only contains contributors, also control the edit links

---
In reply to: [210815815](https://github.com/dotnet/docfx/pull/3224#discussion_r210815815) [](ancestors = 210815815)